R/dataframe-frame.R

```diff
@@ -112,16 +113,55 @@ dataframe__to_struct <- function(name = "") {
     wrap()
 }
 
+#' Convert an existing DataFrame to a LazyFrame
+#' @description Start a new lazy query from a DataFrame.
+#'
+#' @inherit as_polars_lf return
+#' @examples
+#' pl$DataFrame(iris)$lazy()
```

Comment:
The behavior of `pl$DataFrame(<data.frame>)` has been changed.
Could you copy Python example?
https://github.com/pola-rs/polars/blob/577cd62d30361a0e03a018e8761aae7dceedd784/py-polars/polars/dataframe/frame.py#L8884-L8919